meteor/imports/themes/classic/mozdef.css

```diff
@@ -57,6 +57,10 @@ body {
     color: black;
 }
 
+.container {
```

Comment:
I think instead of adding style to the container class that is used by other areas of the web ui, we instead probably want to create a specific id. Currently this will cause headers to be pushed down the page:
Before:
<img width="670" alt="Screen Shot 2019-11-21 at 10 46 34 AM" src="https://user-images.githubusercontent.com/2117578/69358305-41404400-0c4c-11ea-8302-e82a421bbaa3.png">

After:
<img width="631" alt="Screen Shot 2019-11-21 at 10 46 25 AM" src="https://user-images.githubusercontent.com/2117578/69358321-47362500-0c4c-11ea-84d7-24bee6acfc2f.png">

Reply:
Decided to go ahead with this change and then we can focus on fixing up specifically the menu bar.